cvelib/cli.py

```diff
@@ -123,14 +124,25 @@ def handle_cve_api_error(func: Callable) -> Callable:
     def wrapped(*args: Any, **kwargs: Any) -> Callable:
         try:
             return func(*args, **kwargs)
-        except CveApiError as exc:
-            error, _, details = str(exc).partition("; returned error: ")
-            click.secho("ERROR: ", bold=True, nl=False)
-            click.echo(error)
-            if details:
-                click.secho("DETAILS: ", bold=True, nl=False)
-                click.echo(details)
-            sys.exit(1)
+        except requests.exceptions.ConnectionError as exc:
```

Comment:
`ConnectionError` is a subclass of `RequestException` and would ultimately be handled by that same else branch as below, no? So do we really need to catch it explicitly?

Reply:
This is just copied from the existing code in cve_api.py. I kept it the same to avoid accidentally introducing regressions - I just moved to the CLI logic so humans get the nice pretty-printed errors and machines get the raw RequestException. But I could try to simplify it if you'd prefer - I think the original intent was to separately handle RequestException subclasses that might not have the e.response attribute.